file:: [Tinoco_Bustamante_1999_How RNA folds.pdf](file://C:/Users/huyin/Nutstore/1/Papers/Tinoco_Bustamante_1999_How RNA folds/Tinoco_Bustamante_1999_How RNA folds.pdf)
file-path:: file://C:/Users/huyin/Nutstore/1/Papers/Tinoco_Bustamante_1999_How RNA folds/Tinoco_Bustamante_1999_How RNA folds.pdf

- In RNA the two levels of folding (secondary and tertiary) can be experimentally separated by the presence or absence of Mg2Ã¡.
  ls-type:: annotation
  hl-page:: 1
  hl-color:: yellow
  id:: 6476d7e9-ea4f-4062-ac3f-efddfd571874
- Secondary structure can be predicted successfully from experimental thermodynamic data on secondary structure elements: helices, loops, and bulges. Tertiary interactions can then be added without much distortion of the secondary structure.
  ls-type:: annotation
  hl-page:: 1
  hl-color:: yellow
  id:: 6476d86a-b367-487d-9c6e-a807cc8404f2
- These observations suggest a folding algorithm to predict the structure of an RNA from its sequence
  ls-type:: annotation
  hl-page:: 1
  hl-color:: green
  id:: 6476d941-dbe3-4eb4-8a66-29027808257c
- thermodynamic data on tertiary structure interactions, and identiÃ†cation and characterization of metal-ion binding sites.
  ls-type:: annotation
  hl-page:: 1
  hl-color:: red
  id:: 6476e05d-bb8f-41dc-b78e-68e70f6cff9a
- This may mean that we need to know one part of the structure at atomic resolution, another part in terms of Watson-Crick base-pairs formed, and the rest of the molecule only approximately.
  ls-type:: annotation
  hl-page:: 2
  hl-color:: green
  id:: 6476ebec-67cb-4d55-b877-c53f3769c9b3
- If base paired regions in an RNA molecule are conserved over many species, it is assumed that they are important in function.
  ls-type:: annotation
  hl-page:: 2
  hl-color:: green
  id:: 6476efbc-67b2-4cb6-a900-bc00d6ffceb4
- If only a single sequence is available for the RNA, secondary structure calculations can predict the basepaired regions. In viral RNAs the most stable calculated helices correlate with regions that are known to have important biological roles, such as internal ribosome entry sites 
  ls-type:: annotation
  hl-page:: 2
  hl-color:: green
  id:: 6476f2c6-aac4-427a-8a97-e34cf4c04e94
- Structural prediction in protein folding is difÃ†cult for two main reasons. First, because it takes 20 different amino acid residues to build a protein molecule. Thus, the number of distinct interactions among these residues is large, depending not only on the nature of the residues (hydrophobic, hydrophilic, polar, etc.), but also on each residue's detailed structure. Second, because the existence of the various secondary structural elements (a-helices, parallel and antiparallel b-sheets, b-turns, random coils, etc.) is contextual, i.e. these elements form and are stable in the context of the rest of the protein, but may not form when they are isolated in solution. The energies that stabilize secondary structural elements in proteins are comparable to the energies involved in tertiary interactions. Thus, the formation of secondary structure depends on the nature of the tertiary folding contacts, and vice versa. This implies that the contributions of secondary and tertiary interactions to the energetic stability of a protein are, in principle, not separable and, therefore, no simple rules may exist to predict the three-dimensional structure of the protein from its sequence.
  ls-type:: annotation
  hl-page:: 2
  hl-color:: yellow
  id:: 647960f4-a9ad-4163-b93d-950ba1a58f7a
- The four bases are very much alike; there are two purines and two pyrimidines that differ only in the placement of carbonyl and amino groups and their interactions are either through hydrogen bonding or base stacking. Each ribose is a Ã†ve-membered ring with only two main conformations. The phosphate groups with their longrange electrostatic interactions might be considered as providing theoretical difÃ†culties not present in protein folding, but they actually simplify the RNA folding problem both theoretically and experimentally. Electrostatic interactions are well understood theoretically, and experimentally the effective charge on the phosphate groups can be controlled by means of the ionic strength of the solvent.
  ls-type:: annotation
  hl-page:: 2
  hl-color:: yellow
  id:: 64796486-7c54-425f-a644-0159f17fad04
- There are only four basic secondary structure elements in RNA (helices, loops, bulges, and junctions). The helices are A-form Watson-Crick duplexes; the loops, bulges and junctions are all non-Watson-Crick regions terminated by one or more helices. Finally, because the energies involved in the formation of secondary structure are larger than those involved in tertiary interactions, secondary structural elements can exist and be stable by themselves. Thus, the energetic contributions of the secondary and tertiary structural elements are separable, and it is possible to treat the energy of tertiary interactions as a perturbation on the energy stabilizing the secondary structure.
  ls-type:: annotation
  hl-page:: 2
  hl-color:: yellow
  id:: 6479661e-3587-4d51-b2fb-05911a56754f
- The different challenges posed by the protein and RNA folding problems can be traced to the distinct ways in which the information contained in the sequence of these two molecular species controls the secondary and tertiary structures. We can think of the information in the amino acid sequence of a protein as branching into secondary and tertiary structural elements and also Ã˜owing between these elements in both directions. In an RNA, in contrast, the information in the sequenceÃ˜ows linearly, and largely in one direction, Ã†rst to the secondary and then to the tertiary structure. An RNA molecule can thus be thought of as possessing a hierarchical structure in which the primary sequence determines the secondary structure which, in turn, determines its tertiary folding, whose formation alters only minimally the secondary structure. A folding algorithm then is simply a description of the information Ã˜ow from the primary to the tertiary structure.
  ls-type:: annotation
  hl-page:: 2
  hl-color:: yellow
  id:: 64796ad8-2f8a-45cd-bd9a-7dac86dc18d4
- A useful algorithm to predict RNA folding need not tell us anything about the folding pathway of an RNA molecule; the hierarchical nature of the algorithm depends on the relative stabilities of the various structures, not the kinetics of folding.
  ls-type:: annotation
  hl-page:: 3
  hl-color:: blue
  id:: 64796cda-8d50-4c1e-a2a2-d0f99cf12302
- Loops and bulges decrease the entropy of the single strand
  ls-type:: annotation
  hl-page:: 3
  hl-color:: red
  id:: 64796e4e-a4b3-4473-9075-412a9e90b13b
- but a single base-pair is never stable in aqueous solution
  ls-type:: annotation
  hl-page:: 3
  hl-color:: red
  id:: 64797007-ee57-4374-924e-375d1e344282
- Hydrogen bonding provides at most 1 kcal molË‡1 of stabilization; this is not enough to counteract the entropy loss of even a hairpin loop of four nucleotides (G37 Ã  Ã¡ 4.5 kcal molË‡1). 
  ls-type:: annotation
  hl-page:: 3
  hl-color:: red
  id:: 647970d1-481c-4d4c-b086-c8d3520cc897
- The rate of hairpin loop formation thus depends on the frequency of collision between complementary bases that initiate a sequence of base-pairs. This rate depends on the effective concentration of one base relative to another, i.e. on the effective relative volume that is accessible to the two bases. The effective relative volume depends, in turn, on the number of nucleotides separating the two bases, so small hairpin loops will form at the shortest times
  ls-type:: annotation
  hl-page:: 3
  hl-color:: red
  id:: 647971e8-c468-4794-9939-372731faf196
- The explanation is that the kinetics of folding favors formation of small hairpin loops for the same reasons that make these loops thermodynamically stable.
  ls-type:: annotation
  hl-page:: 3
  hl-color:: blue
  id:: 647972d3-0b23-4ed5-8732-e5a86e43eb8f
- Small loops have their ends close together; this means that the ends collide more often and do not lose much entropy on closing.
  ls-type:: annotation
  hl-page:: 3
  hl-color:: blue
  id:: 64797311-2ff3-49de-927e-1c2c56dfbf36
- The main conclusion is that the folding of RNA is sequential as well as hierarchical
  ls-type:: annotation
  hl-page:: 3
  hl-color:: blue
  id:: 647973c1-1ba4-4f15-a1fd-f352843ec7f1
- We recognize that folding does not always occur on the path towards the lowest free energy structure. Clearly, during folding, secondary contacts may transiently form that must be undone before other interactions occur that lead the molecule in the direction of the Ã†nal global free energy minimum.
  ls-type:: annotation
  hl-page:: 3
  hl-color:: blue
  id:: 64797786-29b0-4fa3-b9a9-1f1beca59c4b
- As the temperature of an RNA is decreased slowly, the single strand begins to form stem loops. At the higher temperatures (and depending on the salt concentration) there will be multiple species: unfolded, partly folded, and fully folded.
  ls-type:: annotation
  hl-page:: 4
  hl-color:: red
  id:: 64797eec-657a-4a89-b438-73df8f7216b1
- A bulge is distinguished from an internal loop by continuous pairing of the bases Ã˜anking the bulge.
  ls-type:: annotation
  hl-page:: 4
  hl-color:: red
  id:: 647981f8-eb7c-4854-96a9-68b4d938c747
- The main assumption is that the free energies of stems depend on nearest-neighbor sequences of basepairs only, and that free energies of stems and loops are additive. 
  ls-type:: annotation
  hl-page:: 5
  hl-color:: red
  id:: 64798b91-c2a5-427c-92b0-b83f04837b0b
- Formation of tertiary structure When P5abc is part of the entire ribozyme, or part of the P4-P6 domain of the ribozyme, the secondary structure seen is closest to the one with the least stable free energy (G Ã  Ë‡ 22.0 kcal molË‡1) shown in Figure 2(c). The reason for this change is the formation of tertiary structure. The structure of P4-P6 was determined by X-ray diffraction in a crystal (Cate et al., 1996a); a schematic of the P5abc domain is shown in Figure 3. The Ã†ve nucleotide A-rich bulge has been changed to two bulges, a four nucleotide bulge and a single guanine bulge. The guanine base is linked to the P5c stem by a bound Mg 2Ã¡. The P5c stem loop is changed from the tetraloop to a pentaloop closed by only three base-pairs. Two new G A pairs bound to Mg 2Ã¡ are formed. Essentially the same base-pairing scheme seen in Figure 3 was found by phylogenetic analysis (Michel & Westhof, 1990) of many group I introns. Covariation in sequences with the same function, of bases that are Watson Crick compHow RNA Folds 275 lements indicates base-pairing. The secondary structure thus found corresponds to that present in the functional form i.e. the secondary structure in the presence of tertiary structure. Comparison of Figure 3 and 2 shows one example of how the formation of tertiary structure can change the secondary structure. For P5abc the presence of Mg 2Ã¡ and the presence of the rest of the ribozyme causes some of the less stable parts of the secondary structure to rearrange. Two adenine bases in the A-rich bulge interact with a loop in P4; the P5b tetraloop docks with its receptor in J4/J5; the P5c stem loop is connected to the surrounding RNA by Mg 2Ã¡. These tertiary interactions not only change the three-dimensional arrangement of the RNA; they also change the base-pairing. There are many other examples where the base-pairing is not changed. In tRNA the base-pairing in the cloverleaf is not changed when the tRNA is in its folded, functional form. Tertiary folding in tRNA involves formation of base triples and loop-loop base-pairing, but the stems and loops in the secondary structure are not changed.
  ls-type:: annotation
  hl-page:: 5
  hl-color:: purple
  id:: 64799570-f911-4e9e-839b-af5eeb3eb55c
- divalent ions affect the tertiary folding much more than the secondary structure; speciÃ†c metal ion binding occurs.
  ls-type:: annotation
  hl-page:: 6
  hl-color:: blue
  id:: 647996ae-6b91-4679-a2ff-2fdc07398846
- Therefore GU base-pairs, in particular runs of GU pairs, are expected to be sites for binding solvated magnesium ions, Mg(H 2 O) 62Ã¡
  ls-type:: annotation
  hl-page:: 6
  hl-color:: yellow
  id:: 6479982a-8d08-4f50-9d25-caea7c18f691
- We conclude that after the secondary structure forms, metal ions bind to the pre-formed sites; the metal ion is not required for the formation of the site. However, the presence of the ion in the site favors the formation of tertiary structure by reducing electrostatic repulsion of the phosphate ions.
  ls-type:: annotation
  hl-page:: 7
  hl-color:: blue
  id:: 647998fa-508c-46ab-b181-3d9c7402e2fd
- We thus distinguish two types of speciÃ†callybound metal ions: (1) the metal ions that bind to pre-formed sites in secondary structure and do not change the structure of the site signiÃ†cantly; and(2) the metal ions that are part of the tertiary structure site; without the metal ion neither the site nor the tertiary structure exists.
  ls-type:: annotation
  hl-page:: 7
  hl-color:: blue
  id:: 64799952-ec5e-4700-aa0b-af0c8b944c16
- to predict tertiary structure we need the thermodynamic data not for one pseudoknot, but for pseudoknot formation as a function of stem sequence and length, and as a function of loop lengths.
  ls-type:: annotation
  hl-page:: 8
  hl-color:: blue
  id:: 64799ce8-9871-4d5d-83fe-dda173224bd7
- Now we can apply a force to the ends of the RNA, or to any two residues where we can attach handles, to study the unfolding in any environment, in the presence of any ligands, including bound proteins. Obtaining and completing this thermodynamic library of tertiary interactions is essential for RNA structural prediction.
  ls-type:: annotation
  hl-page:: 10
  hl-color:: blue
  id:: 64799f43-7de1-4ed6-803a-57e8aed9707c
- We want to predict the folded form of an RNA well enough to determine and understand, its function.
  ls-type:: annotation
  hl-page:: 10
  hl-color:: blue
  id:: 6479a007-23a2-48a8-89b3-0633b275538d